pkg/webui/console/views/application-collaborator-add/index.js

```diff
@@ -77,6 +77,14 @@ export default class ApplicationCollaboratorAdd extends React.Component {
     getApplicationsRightsList()
   }
 
+  componentDidUpdate (prevProps) {
```

Comment:
Since we will likely use this pattern a lot, it might make sense to HOC this and use it as a decorator. (Not in this PR obviously)